templates/terraform/provider_gen.erb

```diff
@@ -18,6 +18,11 @@ package google
 
 import "github.com/hashicorp/terraform/helper/schema"
 
+
+<% if product_ns == 'Resourcemanager'
+		product_ns = 'ResourceManager'
```

Comment:
Beautiful 🤦‍♂️

Reply:
In a different PR, we might evaluate adding a way to override a product name the same way we do it for properties or resources.

Reply:
You may want to extract the _meaning_ of the if into a concept and add it to `terraform.yaml`. Adding a product specific like this in the template is a major no-no.